cmd/router/main.go

```diff
@@ -41,6 +41,8 @@ import (
 var log = logf.Log.WithName("InferenceGraphRouter")
 
 func callService(serviceUrl string, input []byte, headers http.Header) ([]byte, error) {
+	defer timeTrack(time.Now(), "step", serviceUrl)
+	log.Info("Entering callService", "url", serviceUrl)
```

Comment:
Do we need this at INFO level? the message does not convey a lot info, we can log more useful information like request id header etc.

Reply:
What I have seen so far is that it's helpful fro tracing a lot. I don't see any downside of having it.